packages/@aws-cdk/aws-cognito/lib/user-pool.ts

```diff
@@ -822,6 +812,17 @@ export class UserPool extends UserPoolBase {
 
     this.userPoolProviderName = userPool.attrProviderName;
     this.userPoolProviderUrl = userPool.attrProviderUrl;
+
+    if (props.lambdaTriggers) {
```

Comment:
Any reason you've moved this from its original location? Would prefer to keep it where it originally was.